applications/StructuralMechanicsApplication/custom_response_functions/structural_response_function_factory.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import print_function, absolute_import, division
+
+# importing the Kratos Library
+from KratosMultiphysics import *
+import KratosMultiphysics.StructuralMechanicsApplication
+import structural_response
+
+def CreateResponseFunction(response_id, response_settings, model_part):
+    response_type = response_settings["response_type"].GetString()
+
+    if response_type == "strain_energy":
+        response_function_utility = StructuralMechanicsApplication.StrainEnergyResponseFunction( model_part, response_settings )
+        return structural_response.SimpleResponseFunctionWrapper(response_id, response_settings, response_function_utility, model_part)
```

Comment:
About the term "simple" and "default", I dropped already a comment above. Here, I find it a bit confusing to read here about a "simple" or "default" function wrapper while in some cases below the name of the function is explicitly given - e.g. "MassResponse...".